app/src/main/java/com/elyeproj/sampleloaderview/MainActivity.java

```diff
@@ -31,6 +31,7 @@ private void loadData() {
     }
 
     private void postLoadData() {
+        ((TextView)findViewById(R.id.txt_presidents_title)).setText(getString(R.string.title, 1));
```

Comment:
There's this `postLoadData` added, but without the `resetLoader` side of it in place